### DISCUSSION TITLE
I'd like to get a refund

### DISCUSSION THREAD

**Discussion by Affc06** [aclap-dev/video-downloadhelper#1303] [License & Purchase requests]
### What's your issue?

I'd like to get a refund, I bought the license with the hope of it making the plugin work for a specific website. I only wanted it for that website and it didn't work.

### The 10 first digits (only 10) of your license number or email address

bd82379de2

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I am just another user.

For anybody coming after you to read this -- this is too late to help you -- people should understand what the VDH license does.  It just stops VDH from writing a watermark into certain downloaded videos.  It also removes a timing restriction on Chrome & Edge that prevents more than 1 HLS download every 2 hours.  The VDH license does not unlock any features.  The VDH license does not make it work on web sites that don't work when you don't have a license.  Unlicensed VDH is 99.99% the same as licensed VDH.  The standard advice is to FIRST test drive VDH to see whether it will work on your favorite sites.  Only AFTER you prove that VDH works on your favorite sites should you bother to buy a VDH license.  That saves you the annoyance of making a purchase that you soon have to get refunded.  Credit card companies are quick to charge your account but slow to process refunds.  So even if you paid today, it will be several days before you get your refund, even if Jérôme were to generate your refund right this second.  So it's better to test drive first to verify that VDH works where you want, & put off buying a license until you're sure VDH works where you want.

**Comment by jcfrog** [comment]
@Affc06 sorry the refund is only available the first month after purchase.
